demo.sh

```diff
@@ -674,18 +610,6 @@ transfer_complete() {
     _sync_wallet "$SEND_WLT"
     _sync_wallet "$RCPT_WLT"
 
-    ## accept transfer
-    local accept
-    _subtit "(recipient) accepting transfer"
-    # note: all output to stderr
-    _trace "${RGB[@]}" -d "data${rcpt_id}" accept -w "$RCPT_WLT" \
-        "$rcpt_data/$CONSIGNMENT" >$TRACE_OUT 2>&1
-    accept="$(cat $TRACE_OUT)"
-    [ $DEBUG = 1 ] && echo "$accept"
-    if ! echo "$accept" | grep -q 'Transfer accepted into the stash'; then
-        _die "accept failed (transfer $TRANSFER_NUM)"
-    fi
-
     ## ending situation
```

Comment:
No more consignment verification command; only `accept`.

The real reason is the fact that consignments now are streams parsed by runtime right into stash - and the stash is an abstract object which can't be cloned in memory. Thus, there is no such thing as a "dumb run": the consignment (if valid) will always get into the stash.